en/integrations/plugin-packs/procedures/cloud-azure-web-appservice.md

```diff
@@ -0,0 +1,383 @@
+
+More information about the Host Discovery module is available in the Centreon documentation:
+[Host Discovery](../../../monitoring/discovery/hosts-discovery.html)
+
+### Collected Metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--App-Usage-->
+
+| Metric Name                          | Description                                                                       |
+|:-------------------------------------|:----------------------------------------------------------------------------------|
+| appservice.connections.count         | The number of bound sockets existing in the sandbox                               |
+| appservice.assemblies.current.count  | The current number of assemblies loaded across all AppDomains in this application |
+| appservice.handle.count              | The total number of handles currently open by the app process                     |
+| appservice.thread.count              | The number of threads currently active in the app process                         |
+| appservice.appdomains.count          | The current number of AppDomains loaded in this application                       |
+| appservice.appdomains.unloaded.count | The total number of AppDomains unloaded since the start of the application        |
+
+<!--Cpu-Time-->
+
+| Metric Name                     | Description                                      | Unit    |
+|:--------------------------------|:-------------------------------------------------|:--------|
+| appservice.cpu.consumed.seconds | The amount of CPU consumed by the app in seconds | seconds |
+
+<!--Data-->
+
+| Metric Name               | Description                                          | Unit  |
+|:--------------------------|:---------------------------------------------------- |:------|
+| appservice.data.in.bytes  | The amount of incoming bandwidth consumed by the app | bytes |
+| appservice.data.out.bytes | The amount of outgoing bandwidth consumed by the app | bytes |
+
+<!--File-System-->
+
+| Metric Name                       | Description                                        |
+|:----------------------------------|:---------------------------------------------------|
+| appservice.filesystem.usage.bytes | Percentage of filesystem quota consumed by the app |
+ 
+<!--Gc-Usage-->
+
+| Metric Name              | Description                                                                                           |
+|:-------------------------|:------------------------------------------------------------------------------------------------------|
+| appservice.gc.gen0.count | The number of times the generation 0 objects are garbage collected since the start of the app process |
+| appservice.gc.gen1.count | The number of times the generation 1 objects are garbage collected since the start of the app process |
+| appservice.gc.gen2.count | The number of times the generation 2 objects are garbage collected since the start of the app process |
```

Comment:
```suggestion
| Metric Name              | Description                                                                                           | Unit  |
|:-------------------------|:------------------------------------------------------------------------------------------------------|:------|
| appservice.gc.gen0.count | The number of times the generation 0 objects are garbage collected since the start of the app process | Count |
| appservice.gc.gen1.count | The number of times the generation 1 objects are garbage collected since the start of the app process | Count |
| appservice.gc.gen2.count | The number of times the generation 2 objects are garbage collected since the start of the app process | Count |
```